app.py

```diff
@@ -62,8 +64,25 @@ def delete(todo_id):
     return redirect(url_for("index"))
 
 
+@app.route('/complete',methods=["GET"])
+def complete():
+    # filter done and not done
+    todo = Todo_List.query.filter_by(done=True).all()
+    print(todo)
+    return render_template("base.html",todo=todo)
 
 
+@app.route('/incomplete',methods=["GET"])
+def incomplete():
```

Comment:
This is a very OK way to achieve the filter functionality but the most ideal way to do it is via the query parameter i.e GET /todo?type=complete.